libcxx/include/__math/hypot.h

```diff
@@ -41,6 +46,83 @@ inline _LIBCPP_HIDE_FROM_ABI typename __promote<_A1, _A2>::type hypot(_A1 __x, _
   return __math::hypot((__result_type)__x, (__result_type)__y);
 }
 
+#if _LIBCPP_STD_VER >= 17
+template <class _Real>
+struct __hypot_factors {
+  _Real __threshold;
+  _Real __scale_xyz;
+  _Real __scale_M;
+};
+
+// returns [underflow_factors, overflow_factors]
+template <class _Real>
+std::array<__hypot_factors<_Real>, 2> __create_factors() {
```

Comment:
```suggestion
_LIBCPP_HIDE_FROM_ABI array<__hypot_factors<_Real>, 2> __create_factors() {
```

Reply:
This might be a newbie question, but what is the purpose of removing `std::` for `std::array`?

Reply:
It's just as valid, it's only a matter of consistency. We generally don't qualify types when we don't need to throughout the library (not with 100% consistency though).

Reply:
Fixed in ada9edbb84e415b443fb23bc792effcba873762f, aeb72521a887bb2eed14ec4f8e907314a936aaef.